lib/fog/aws/credential_fetcher.rb

```diff
@@ -44,7 +42,15 @@ def fetch_credentials(options)
                   :WebIdentityToken => File.read(options[:aws_web_identity_token_file] || ENV.fetch("AWS_WEB_IDENTITY_TOKEN_FILE")),
                   :Version => "2011-06-15",
                 }
-                connection = options[:connection] || Excon.new(STS_GLOBAL_ENDPOINT, :query => params)
+
+                sts_endpoint =
+                  if ENV["AWS_STS_REGIONAL_ENDPOINTS"] == "regional" && region
```

Comment:
Part of this still seems a little weird to me. I guess it feels like maybe the key should just be something like `true`? ie `ENV["AWS_STS_REGIONAL_ENDPOINTS"]` is binary, so making it that explicitly seems a bit cleaner to me than using a special `regional` key. Does that make sense?

Reply:
Ah, wait, I see. It matches the keys/values from the AWS CLI. That makes more sense.